build.rs

```diff
@@ -43,6 +43,8 @@ fn main() -> Result<(), Box<dyn Error>> {
     match &target[..] {
         "avr-unknown-gnu-atmega328"
         | "msp430-none-elf"
+        | "avr-atmega328p"
```

Comment:
should be `avr-unknown-gnu-atmega328`

Reply:
I have discovered that I can just remove these match clauses, so I am.